xml/snmp_sysdescr.xml

```diff
@@ -1079,6 +1082,47 @@
     <param pos="2" name="os.version"/>
   </fingerprint>
 
+  <fingerprint pattern="^Brocade VDX Switch, BR-(\S+), Network Operating System Software Version (\S+)\.$">
+    <description>Brocade VDX Switch</description>
+    <example hw.product="VDX6740" os.version="7.0.2b">Brocade VDX Switch, BR-VDX6740, Network Operating System Software Version 7.0.2b.</example>
+    <param pos="0" name="os.vendor" value="Brocade"/>
+    <param pos="0" name="os.device" value="Switch"/>
+    <param pos="0" name="os.family" value="NetworkOS"/>
+    <param pos="2" name="os.version"/>
+    <param pos="0" name="hw.vendor" value="Brocade"/>
+    <param pos="0" name="hw.family" value="VDX"/>
+    <param pos="1" name="hw.product"/>
```

Comment:
I moved these Rumble related fingerprints (from the recently landed PR #334 ) in order to group them with the other Brocade fps. Unfortunately, this hid the changes I made.  They are:
1. Adding `hw.*` fields
2. Moving `os.product` to `hw.product`.

CC @hdm in the event that this is a breaking change for Rumble.

Reply:
👍